examples/fullstack/backend/lib/functions.dart

```diff
@@ -24,14 +24,17 @@ export 'api_types.dart';
 @CloudFunction()
 GreetingResponse function(GreetingRequest request, RequestContext context) {
   final name = request.name ?? 'World';
-  final response = GreetingResponse(salutation: getSalutation(), name: name);
+  final response =
+      GreetingResponse(salutation: _randomSalutation(), name: name);
   context.logger.info('greetingResponse: ${response.toJson()}');
   return response;
 }
 
-final Random rng = Random(DateTime.now().millisecondsSinceEpoch);
+String _randomSalutation() => salutations[_random.nextInt(salutations.length)];
 
-final List<String> salutations = [
+final _random = Random();
```

Comment:
Why no seed? Does the Dart implementation apply a default seed to ensure output isn't the same for each program invocation?

Reply:
I wish the docs were better here, but yet. If `seed` isn't provided, a timer-based value is used.

Reply:
Ah, thanks for the explanation. Cool, then I suppose adding a seed is only for when you want "reproducible" randomness for testing, or you know you really do want a special seed.